ingestion/ingestion_dependency.sh

```diff
@@ -39,8 +39,8 @@ airflow users create \
     --email spiderman@superhero.org \
     --password ${AIRFLOW_ADMIN_PASSWORD}
 
-(sleep 5; airflow db upgrade)
-(sleep 5; airflow db upgrade)
+(sleep 5; airflow db migrate)
```

Comment:
modified `airflow db upgrade` to `airflow db migrate`

as Prior to Airflow version 2.7.0, airflow db upgrade was used to apply migrations, however, it has been deprecated in favor of airflow db migrate.


Ref: https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/production-deployment.html#database-backend